src/main/resources/templates/fragments/userBar.html

```diff
@@ -9,32 +9,21 @@
   <nav class="content">
     <ul class="mobile-hidden" id="navigation">
       <li>
-        <ul class="mobile-hidden"
-            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
+        <ul th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+            th:remove="tag">
           <li class="user" id="signed-in-user" th:text="${userModel.getId() + ':'}"></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your details - top menu" id="your-details"
-              th:href="@{{chsAccountUrl}/user/account(chsAccountUrl=${@environment.getProperty('account.url')})}">Your
-            details
-          </a></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your Applications" id="your-applications"
-              th:href="@{{manageApplicationsUrl}/(manageApplicationsUrl=${@environment.getProperty('manageApplications.url')})}">Your
-            Applications
-          </a></li>
           <li><a class="piwik-event" data-event-id="Sign out" href="/signout" id="user-signout">Sign
             out
           </a></li>
         </ul>
       </li>
-      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-        <!--  -->
+      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+          th:remove="tag">
         <a class="piwik-event govuk-link" data-event-id="Sign In" href="/signin" id="user-signin">Sign
           in /
           Register</a>
       </li>
+      <li><a class="govuk-link" href="https://forum.aws.chdev.org/">Developer hub forum</a></li>
```

Comment:
Please see above comment - Michael Gearon suggested we should have the dev hub forum link before Sign Out link. This mirrors current live developer hub.

Reply:
Changed.